src/app/shared/components/template/components/text-bubble/text-bubble.component.ts

```diff
@@ -8,7 +8,9 @@ interface ITextBubbleParams {
   /** TEMPLATE PARAMETER: "speaker_position". The position of the speaker image and speech bubble tail */
   speakerPosition: "left" | "right";
   /** TEMPLATE PARAMETER: "variant" */
-  variant: "gray" | "primary" | "secondary" | "no-border";
+  variant: "gray" | "primary" | "secondary" | "no-border" | "speaker-3" | "speaker-4";
+  /** TEMPLATE PARAMETER: "avatar_name". The name of the speaker */
```

Comment:
nit(non-blocking): I assume this comment can also be updated now to `speaker_name`.

Also not sure if we're really using a `name` or not here. Looking at the variants above I'm guessing we're more using a number to identify which speaker within a list of potential speakers that have different styling. If so might make more sense to use `speaker_number`. But I don't feel super strongly either way